specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2022-08-11-preview/HybridCompute.json

```diff
@@ -0,0 +1,1637 @@
+          "$ref": "#/definitions/OperationValueDisplay",
+          "description": "Display properties"
+        },
+        "isDataAction": {
+          "type": "boolean",
+          "readOnly": true,
+          "description": "This property indicates if the operation is an action or a data action"
+        }
+      },
+      "description": "Describes the properties of a Compute Operation value."
+    },
+    "OperationValueDisplay": {
+      "type": "object",
+      "properties": {
+        "operation": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The display name of the compute operation."
+        },
+        "resource": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The display name of the resource the operation applies to."
+        },
+        "description": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The description of the operation."
+        },
+        "provider": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The resource provider for the operation."
+        }
+      },
+      "description": "Describes the properties of a Hybrid Compute Operation Value Display."
+    },
+    "PatchSettings": {
+      "type": "object",
+      "properties": {
+        "assessmentMode": {
+          "type": "string",
+          "description": "Specifies the assessment mode.",
+          "enum": [
+            "ImageDefault",
+            "AutomaticByPlatform"
+          ],
+          "x-ms-enum": {
+            "name": "AssessmentModeTypes",
+            "modelAsString": true
+          }
+        },
+        "patchMode": {
+          "type": "string",
+          "description": "Specifies the patch mode.",
+          "enum": [
+            "ImageDefault",
+            "AutomaticByPlatform",
+            "AutomaticByOS",
+            "Manual"
+          ],
+          "x-ms-enum": {
+            "name": "PatchModeTypes",
+            "modelAsString": true
+          }
+        }
+      },
+      "description": "Specifies the patch settings."
+    },
+    "AgentConfiguration": {
+      "type": "object",
+      "readOnly": true,
+      "properties": {
+        "proxyUrl": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Specifies the URL of the proxy to be used."
+        },
+        "incomingConnectionsPorts": {
+          "type": "array",
+          "readOnly": true,
+          "description": "Specifies the list of ports that the agent will be able to listen on.",
+          "items": {
+            "type": "string"
+          }
+        },
+        "extensionsAllowList": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/ConfigurationExtension"
+          },
+          "description": "Array of extensions that are allowed to be installed or updated.",
+          "x-ms-identifiers": []
```

Comment:
>[] [](http://example.com/codeflow?start=30&length=2)

pl add identifiers wherever it makes sense

Reply:
Hi @rkmanda. These are not related to our current set of updates to the spec.

We can take these as items for our next version update.

Currently delayed due to the slow progress of this spec review.